spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,162 @@
+require 'rails_helper'
+
+RSpec.describe ReviewsController, type: :controller do
+  describe "GET index" do
+    it "has a 200 status code" do
+      get :index
+      expect(response.status).to eq(200)
+    end
+  end
+
+  describe '#create' do
+    let!(:rest) { FactoryBot.create(:restaurant) }
+    it "creates a new restaurant" do
+      post :create, params: { restaurant_id: rest.id,
+                              reviewer_name: "Dotan Langsam Schwartz",
+                              rating: 3,
+                              comments: 'I like it here'}
+      expect(response.status).to eq(200)
+
+      review_id = JSON.parse(response.body)['id']
+      expect(Review.where(:restaurant_id => rest.id).count()).to eq(1)
+      expect(Review.find_by_restaurant_id(rest.id)['id']).to eq(review_id)
+    end
+
+    it "can't create a review without a restaurant_id" do
+      post :create, params: { reviewer_name: "Dotan Langsam Schwartz",
+                              rating: 3,
+                              comments: 'I like it here'}
+      expect(response.status).to eq(400)
+      expect(Review.count).to eq 0
+    end
+
+    it "creates a review without optional fields" do
+      post :create, params: { restaurant_id: rest.id,
+                              reviewer_name: "Dotan Langsam Schwartz",
+                              rating: 3}
+      expect(response.status).to eq(200)
+
+      review_id = JSON.parse(response.body)['id']
+      expect(Review.where(:restaurant_id => rest.id).count()).to eq 1
+      expect(Review.find_by_restaurant_id(rest.id)['id']).to eq(review_id)
+    end
+
+    it "can't create a review without rating" do
```

Comment:
You could frame these type of test under illegal inputs. Wrap them with something like:
`context 'illegal input' do
   let(:rest) {FactoryBot.build(:review, :missing_args)
end`

Scoping adds with readability in the manner of a knowing where to search for what, and having common args for common behaviour tests.